### ISSUE TITLE
Dragend stopped working on desktop browsers (Chrome 39.0.2171.95 and Firefox)

### ISSUE THREAD

**Issue by ndkv** [Stereobit/dragend#48]
Dragend doesn't work on Chrome anymore. 

It seems that the `isTouch = 'ontouchstart' in win` check on [line 100](https://github.com/Stereobit/dragend/blob/master/dragend.js#L100) isn't valid anymore. `ontouchstart` is present in `win` (but has a value of null) hence `isTouch` always evaluates to `true`. This breaks swiping on desktop browsers as mouse events are not handled anymore. 

This was already the case with Firefox (see my brief investigation in issue #46) but is now also happening on Chrome. Last week everything worked fine on Chrome (desktop) as it didn't have an `ontouchstart` property in `win` and the check on line 100 returned `false`.

Oddly enough, mobile browsers also report that `ontouchstart` is `null` so I wouldn't know how to differentiate between desktop and mobile without using a full-fledged browser detection library. Any thoughts?

**Comment by Stereobit** [comment]
Hey @ndkv, thanks for the investigation. Since ```null``` is a falsy value - would changing it to ```win.ontouchstart``` help?

**Comment by ndkv** [comment]
I don't think so as ```ontouchstart``` is also ```null``` on mobile browsers. Simply checking its value will break dragend on them.

Detecting touch is a messy business (see this [Stackoverflow question](http://stackoverflow.com/questions/4817029/whats-the-best-way-to-detect-a-touch-screen-device-using-javascript)).

Seeing that these things apparently change overnight, I think the best strategy is to include a [custom build of e.g. Modernizr](http://modernizr.com/download/#-fontface-backgroundsize-borderimage-borderradius-boxshadow-flexbox-hsla-multiplebgs-opacity-rgba-textshadow-cssanimations-csscolumns-generatedcontent-cssgradients-cssreflections-csstransforms-csstransforms3d-csstransitions-applicationcache-canvas-canvastext-draganddrop-hashchange-history-audio-video-indexeddb-input-inputtypes-localstorage-postmessage-sessionstorage-websockets-websqldatabase-webworkers-geolocation-inlinesvg-smil-svg-svgclippaths-touch-webgl-shiv-cssclasses-addtest-prefixed-teststyles-testprop-testallprops-hasevent-prefixes-domprefixes-load) to check for touch capabilities.

**Comment by gebrits** [comment]
Thinking of using this lib, but this issue makes me hesitant. Any progress?
